.github/actions/setup-fluvio/action.yml

```diff
@@ -0,0 +1,25 @@
+name: Setup Fluvio
+description: "Download and install Fluvio CLI via FVM"
+inputs:
+  version:
+    description: "Fluvio version to install"
+    required: false
+    default: "stable"
+
+runs:
+  using: "composite"
+  steps:
+    - name: Check Platform Compatibility
+      if: runner.os != 'Linux' && runner.os != 'macOS'
+      shell: bash
+      run: |
+        echo "Current platform $env:RUNNER_OS, is not supported by Fluvio CLI Installer."
+        echo "Please use a Linux or macOS runner."
+        echo "For further details on Fluvio Installation, please visit https://fluvio.io/docs/install/"
+        exit 1
+
+    - name: Install Fluvio
+      shell: bash
+      run: |
+        curl -fsS https://hub.infinyon.cloud/install/install.sh | FLUVIO_VERSION=${{ inputs.version }} bash
```

Comment:
The url needs ctx-ci

Reply:
Here its added: [3f9d78c](https://github.com/infinyon/fluvio/pull/3775/commits/3f9d78c8f55043ce3350ad890c6acdb3a9519fa2)! Thanks!